mantle/cmd/kola/options.go

```diff
@@ -168,6 +168,18 @@ func syncOptionsImpl(useCosa bool) error {
 		return err
 	}
 
+	if kola.Options.CosaBuild == "" {
+		if err := sdk.ValidateCosaRoot("."); err == nil {
```

Comment:
Can we explicitly check in the `else` path that the error was `ENOENT` and error out otherwise?